modules/social_features/social_user/tests/src/Kernel/GraphQL/GraphQLViewerEndpointTest.php

```diff
@@ -25,8 +26,8 @@ class GraphQLViewerEndpointTest extends SocialGraphQLTestBase {
    * It loads the current user.
    */
   public function testViewerLoadsCurrentUser() : void {
-    $user = $this->createUser();
-    $this->setCurrentUser($user);
+    $user = $this->setUpCurrentUser();
+    self::assertInstanceOf(UserInterface::class, $user, "Test setup failure: could not create test user.");
```

Comment:
setUpCurrentUser creates a user and returns it. However, it can also return NULL so we must still add the assertion similarly as we've done for `Server::load` so that our typechecker can keep track of what type it actually is. 

If the assertion fails that'd be a bug in the `setUpCurrentUser` which should never happen. Then again, we're programmers, so we're paid to guard against the things that should never happen (but still do) :D